docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Docstrings should use double- not single-quotes according to PEP8, and we use Google Styled docstrings in our codebases, not sphinx style, so I'd prefer to follow that here:</p><p>https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings</p><p> </p><p>Also type hints would be nice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/hyp3-docs/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>